bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/BigtableHBaseSettings.java

```diff
@@ -75,6 +75,12 @@ public String getInstanceId() {
 
   public abstract long getBatchingMaxRequestSize();
 
+  /**
+   * This represents settings to allows mutation retries without timestamp. This is equivalent to
+   * allow server-side timestamp.
+   */
+  public abstract boolean allowRetriesWithoutTimestamp();
+
```

Comment:
Method name sounds confusing to me, I would think that calling it will allow doing retries.
It's better to name it:
isRetriesWithoutTimestampAllowed()

javadoc for method should start with a verb. Could you consider rephrase or remove it (other similar methods do not have such comments)

Reply:
Thanks for the review!!

 I would update the method name and would try to come up with a short description for other method but most of these are self-explanatory that's why I left them out.